lldb/test/CMakeLists.txt

```diff
@@ -267,6 +267,21 @@ add_lit_testsuite(check-lldb "Running lldb lit test suite"
     lldb-shell-test-deps
     lldb-unit-test-deps)
 
+# This target covers all targets that are tied to implementation details
+# of libc++, intended to be run by the libc++ pre-merge CI.
+add_lit_testsuite(check-lldb-libcxx-integration "Running lldb libc++ support test suite"
```

Comment:
As part of this patch, you could also change https://github.com/llvm/llvm-project/blob/main/libcxx/utils/ci/run-buildbot#L374 to call this target, that way everything is already plumbed together!

Reply:
good idea, done!